lib/discourse_event.rb

```diff
@@ -17,7 +17,7 @@ def self.trigger(event_name, *params)
 
   def self.on(event_name, &block)
     if event_name == :site_setting_saved
-      Discourse.deprecate("The :site_setting_saved event is deprecated. Please use :site_setting_changed instead", since: "2.3.0beta8", drop_from: "2.4")
+      Discourse.deprecate("The :site_setting_saved event is no longer supported. Please use :site_setting_changed instead")
```

Comment:
I think we should remove that warning completely. We don't want to keep it indefinitely…

Reply:
No, but if something is suddenly broken people might check the logs for clues. How about making it an error and removing it entirely after the next release?

Reply:
Raising an error might be a good idea, how about this:

```suggestion
      Discourse.deprecate("The :site_setting_saved event is deprecated. Please use :site_setting_changed instead", since: "2.3.0beta8", drop_from: "2.4", raise_error: true)
```